packages/trader/src/AppV2/Components/OrderDetails/order-details.scss

```
@@ -0,0 +1,28 @@
+.order-details {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: var(--component-modal-bg);
+    border-radius: var(--semantic-borderRadius-md);
+
+    &__table {
+        display: flex;
+        justify-content: space-between;
+        flex-direction: column;
+
+        &-row {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+            border-bottom: 1px solid var(--core-color-opacity-black-100);
```

```suggestion
            border-block-end: var(--core-borderWidth-75) solid var(--core-color-opacity-black-100);
```
since we agreed to use logical CSS properties :)

---

Re-inventing the wheel is what I love about this job, but I doubt the bottom will ever be flipped to the top.